cdci_data_analysis/__init__.py

```diff
@@ -24,6 +24,6 @@
 with open('%s/pkg_info.json'%_dir) as fp:
     _info = json.load(fp)
 
-__version__ = _info['version']
+__version__ = _info['version'] + os.getenv('DISPATCHER_EXTRA_VERSION', '')
```

Comment:
What does the `DISPATCHER_EXTRA_VERSION` env var contain?

Because it might not be distinguishable between that and the version. Better to add a space in between.

Reply:
space is no good, this better be a word. the variable would also contain the separator, e.g. a dash.
But I changed a bit the approach now.

Reply:
actually I intended a separator like a dash, space is not good indeed

Reply:
where is the `DISPATCHER_EXTRA_VERSION` var set?

Reply:
in the deployment